### PR TITLE
add option to add id and name to select template

### PR DESCRIPTION
**Pull request by hippogriffic**
@incuna/frontend please review

### CODE REVIEW

**Comment by wytrych** [comment]
Cool. You might want to wait with release as you might discover you need that for something else too

**Comment by hippogriffic** [comment]
good thinking, I will use the commit for now

**Comment by hippogriffic** [comment]
actually it would probably be a good idea to add this option to all the field types... someday

**Comment by hippogriffic** [comment]
uh oh @wytrych travis is angry :(

**Comment by wytrych** [comment]
Oh wow that's bizzare

**Comment by wytrych** [comment]
It might be that there were some changes in the newest angular? I'll take a look, you can still use the commit in your project though even if unmerged

**Comment by hippogriffic** [comment]
ok thanks

File: src/templates/aif/select-input/template.html [hunk 1]
```diff
@@ -3,7 +3,10 @@
         class="select"
         ng-model="model"
         ng-disabled="disable"
-        ng-options="choice.value as choice.display_name for choice in choices">
+        ng-options="choice.value as choice.display_name for choice in choices"
+        id="{{ field-id }}"
+        name="{{ field-id }}"
```
Comment (wytrych) [inline src/templates/aif/select-input/template.html]: It should be `fieldId` in both cases

**Comment by pandalion** [comment]
@wytrych can this be merged? :)

**Pull request merged**